queries.sql

```diff
@@ -15,5 +15,28 @@ SELECT name FROM animals WHERE name <> 'Gabumon';
 
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
+-- Day two
+
+UPDATE animals
+SET species = CASE
+WHEN name LIKE '%mon' THEN 'digimon'
+ELSE 'pokemon'END
```

Comment:
* As stated in the requirement 

> verify that the change was made after updating animals' table 

> verify that change persists after commit, Kindly work on these 🙏

Reply:
Thanks @Davidosky007 for reviewing this PR!

Reply:
Here! I did all the changes suggested by the previous reviewer!